src/annotator/guest.js

```diff
@@ -550,21 +636,38 @@ export default class Guest extends Delegator {
     }
   }
 
-  selectAnnotations(annotations, toggle) {
+  /**
+   * Set the selected annotations in the sidebar.
+   *
+   * @param {AnnotationData[]} annotations
+   * @param {boolean} [toggle]
+   */
+  selectAnnotations(annotations, toggle = false) {
     if (toggle) {
       this.toggleAnnotationSelection(annotations);
     } else {
       this.showAnnotations(annotations);
     }
   }
 
-  // Did an event originate from an element in the annotator UI? (eg. the sidebar
-  // frame, or its toolbar)
+  /**
+   * Did an event originate from an element in the annotator UI? (eg. the sidebar
+   * frame, or its toolbar)
```

Comment:
Wait...does this include events in the guest document or only in the sidebar? I think this comment could use some clarification... the method name involves `Annotator`, so it's a little unclear.